### PR TITLE
add the fee to the type of CosmosSignTx

### PR DESCRIPTION
**Pull request by SterL**

### CODE REVIEW

**Review by BitHighlander — APPROVED** [review]
:shipit:

**Review by majorhayes — APPROVED** [review]

File: packages/hdwallet-core/src/cosmos.ts [hunk 1]
```diff
@@ -56,6 +56,7 @@ export interface CosmosTx {
 
 export interface CosmosSignTx {
   addressNList: BIP32Path;
+  fee?: number;
```
Comment (natven) [inline packages/hdwallet-core/src/cosmos.ts:59]: I like listing optionals last. not a big deal

**Review by majorhayes — APPROVED** [review]

File: packages/hdwallet-core/src/cosmos.ts (hunk 1 above)
Comment (SterL) [reply packages/hdwallet-core/src/cosmos.ts]: coo :+1: fixed

**Review by BitHighlander — APPROVED** [review]
🚢

**Pull request merged**